src/libs/portfolio/helpers.ts

```diff
@@ -347,6 +348,32 @@ export const tokenFilter = (
   return isInAdditionalHints || pinnedRequested
 }
 
+const getTokenSafetyLevel = (token: TokenResult) => {
+  let safetyLevel = 'unknown' as 'unknown' | 'trusted' | 'spoof'
+
+  if (token.address === ZeroAddress) {
+    return 'trusted'
+  }
+
+  Object.values(humanizerInfo.knownAddresses).forEach((value) => {
```

Comment:
kolega, this code is quote hard to read as-is, because you have to follow iteration

can't you do
```
const matchingEntry = humanizerInfo.knownAddresses.find(...)
if (matchingEntry) {
   // found in humanizer
   const isTrusted  = ...
}
```

but also I don't get the logic in general, why would the same address token have a different symbol? I think it would be better to search the humanizer entries for ones that have the same exact symbol but diff address, and in that case mark it as spoof

Reply:
I've written it this way because the humanizer isn't divided by networks.